test/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/generators/ClaimsAnswerGen.scala

```diff
@@ -19,11 +19,11 @@ package uk.gov.hmrc.cdsreimbursementclaimfrontend.models.generators
 import org.scalacheck.Gen
 import org.scalacheck.ScalacheckShapeless._
 import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.Claim
-import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.ClaimsAnswer.CompleteClaimsAnswer
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.ClaimsAnswer
 
 object ClaimsAnswerGen extends GenUtils {
 
-  implicit val claimGen: Gen[Claim]                               = gen[Claim]
-  implicit val completeClaimsAnswerGen: Gen[CompleteClaimsAnswer] = gen[CompleteClaimsAnswer]
+  implicit val claimGen: Gen[Claim]                       = gen[Claim]
+  implicit val completeClaimsAnswerGen: Gen[ClaimsAnswer] = gen[ClaimsAnswer]
```

Comment:
Lets rename `completeClaimsAnswer` to `claimsAnswer`

Reply:
Done